common/rya.api/src/main/java/org/apache/rya/api/instance/ConfigurationFields.java

```diff
@@ -31,5 +31,7 @@
     static final String USE_FREETEXT = "sc.use_freetext";
     static final String USE_TEMPORAL = "sc.use_temporal";
     static final String USE_ENTITY = "sc.use_entity";
-    static final String USE_PCJ = "sc.use_pcj";
```

Comment:
I removed this because sc.use.pcj is the incorrect flag to use when overwriting user config.  This flag corresponds to whether a query client should use pcj, not whether or not to use the PrecomputedJoinIndex (i.e. Fluo).  Originally this was the correct flag to use for the Fluo Updater, but additional config parameters were added to separate the config for client PCJ use and the creation of the PrecomputedJoinIndex.